source/features/sticky-comment-header.css

```
@@ -17,6 +22,8 @@ html:not([rgh-OFF-sticky-comment-header]) {
 			var(--base-sticky-header-height, 0px) +
 			var(--rgh-issue-or-pr-sticky-header-height, 0px)
 		);
+		/* Fallback in case `show-names` is disabled - #8726#issuecomment-3606420543 */
```

```suggestion
		/* Fallback in case `show-names` is disabled */
		/* https://github.com/refined-github/refined-github/pull/8726#issuecomment-3606420543 */
```

Due to https://github.com/refined-github/refined-github/issues/6336 I reduced my reliance on references in code